### PR TITLE
[Feature] Add scheduler visualization from mmpretrain to mmocr

### PR DESCRIPTION
**Pull request by A-new-b**
## Motivation

Transfer the scheduler visualization tool from mmpretrain to mmocr.

## Modification

1. create /tools/visualizations/vis_scheduler.py.
2. move /tools/analysis_tools/browse_dataset.py to /tools/visualizations/browse_dataset.py.
3. modify /docs/zh_cn/user_guides/useful_tools.md and /docs/en/user_guides/useful_tools.md for scheduler visualization tool.

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmocr/blob/main/.github/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmocr/blob/main/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/zh_cn/user_guides/useful_tools.md [hunk 1]
```diff
@@ -85,11 +85,58 @@ icdar2015_1811_textrecog_test = dict(
...
+```{note}
+部分数据集在解析标注阶段比较耗时，可直接将 `-d, dataset-size` 指定数据集的大小，以节约时间。
+```
+
+#### 如何在开始训练前可视化学习率曲线
+
+你可以使用如下命令来绘制配置文件 `configs/textdet/dbnet/dbnet_resnet50-dcnv2_fpnc_1200e_icdar2015.py` 将会使用的变化率曲线：
+
+```bash
+python tools/visualizations/vis_scheduler.py configs/textdet/dbnet/dbnet_resnet50-dcnv2_fpnc_1200e_icdar2015.py
```
Comment (gaotongxiao) [inline docs/zh_cn/user_guides/useful_tools.md:133]: Sometimes users may not have the dataset ready but still want to visualize the curve. We can add `-d 100` to this sample command to bypass the dataset loading process (Same for the English docs)

Comment (A-new-b) [reply docs/zh_cn/user_guides/useful_tools.md:133]: Do you mean I should add the -d100 use case?

Comment (gaotongxiao) [reply docs/zh_cn/user_guides/useful_tools.md:133]: Yes, we can only leave the `-d 100` use case as the default example. For this example, if users haven't got icdar2015 dataset ready locally, they will run into an error.

**Comment by gaotongxiao** [comment]
And the linting test seems not passed. 

image

You can install pre-commit hooks to automatically fix this issue:
```
pip install -U pre-commit
pre-commit install
pre-commit run --all-files
git add --update
git commit
```

**Comment by A-new-b** [comment]
> And the linting test seems not passed.
> 
> image
> 
> You can install pre-commit hooks to automatically fix this issue:
> 
> ```
> pip install -U pre-commit
> pre-commit install
> pre-commit run --all-files
> git add --update
> git commit
> ```
This is because I've directly committed your changes, I'll fix it later

**Review by gaotongxiao — APPROVED** [review]
LGTM

**Pull request merged**